README.md

```diff
@@ -0,0 +1,75 @@
+Stitch
```

Comment:
This file can be removed though; The docs are built with sphinx and restructured text.